supervisor/shared/board_busses.c

```diff
@@ -55,19 +58,20 @@ STATIC mp_obj_t board_i2c(void) {
     return i2c_singleton;
 }
 #else
-STATIC mp_obj_t board_i2c(void) {
+mp_obj_t board_i2c(void) {
     mp_raise_NotImplementedError(translate("No default I2C bus"));
     return NULL;
 }
 #endif
 MP_DEFINE_CONST_FUN_OBJ_0(board_i2c_obj, board_i2c);
 
 #if BOARD_SPI
+STATIC busio_spi_obj_t spi_obj;
```

Comment:
Why is this a `static` but the other ones are on the heap?

Reply:
The spi object needs to live longer than the heap because it may be used by a FourWire between VM runs.

Reply:
Can you just add a comment about that? Thanks.